infra/resources/modules/commons/eventhub.tf

```
@@ -0,0 +1,23 @@
+module "subscription_requests_event_hub" {
```

We can create a common event hub without specifying topic purpose
```suggestion
module "event_hub" {
```